Configuration/TSconfig/ContentElementWizard.tsconfig

```diff
@@ -1,16 +1,79 @@
-mod.wizards.newContentElement.wizardItems.plugins {
+mod.wizards.newContentElement.wizardItems.ext-service-bw2 {
```

Comment:
This extension now just supports TYPO3 12. So please migrate that to page.tsconfig